forge-core/src/main/java/forge/deck/Deck.java

```diff
@@ -125,12 +123,7 @@ public List<PaperCard> getCommanders() {
             result.add(c.getKey());
         }
         if (result.size() > 1) { //sort by type so signature spell comes after oathbreaker
-            Collections.sort(result, new Comparator<PaperCard>() {
-                @Override
-                public int compare(final PaperCard c1, final PaperCard c2) {
-                    return Boolean.compare(c1.getRules().canBeSignatureSpell(), c2.getRules().canBeSignatureSpell());
-                }
-            });
+            Collections.sort(result, (c1, c2) -> Boolean.compare(c1.getRules().canBeSignatureSpell(), c2.getRules().canBeSignatureSpell()));
```

Comment:
would `Comperator.comparing(c -> c.getRules().canBeSignatureSpell())` work?

Reply:
Yeah, that looks better.